core/services/runs.go

```diff
@@ -86,6 +98,22 @@ func NewRun(
 	run.CreationHeight = models.NewBig(currentHeight)
 	run.ObservedHeight = models.NewBig(currentHeight)
 
+	if !MeetsMinimumPayment(job.MinPayment, input.Amount) {
+		logger.Debugw("Rejecting run with insufficient payment", []interface{}{
```

Comment:
I'd probably bounce this up to `Infow`, it would be useful to see in the logs in production.